flink-runtime/src/test/java/org/apache/flink/runtime/rest/FileUploadHandlerTest.java

```diff
@@ -256,6 +257,14 @@ public void testUploadCleanupOnFailure() throws IOException {
 		verifyNoFileIsRegisteredToDeleteOnExitHook();
 	}
 
+	private OkHttpClient createOkHttpClientWithNoTimeouts() {
+		return new OkHttpClient().newBuilder()
```

Comment:
maybe use `new OkHttpClient.Builder()...`, it's a bit odd that we are creating 2 clients?

Reply:
True. Will update the PR.